externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/kyuubi/SparkConsoleProgressBar.scala

```diff
@@ -81,7 +93,11 @@ class SparkConsoleProgressBar(
     val width = TerminalWidth / stages.size
```

Comment:
Now we only display the progress information of three stages. 
`show(now, stages.take(3)) // display at most 3 stages in same time`
After adding Job information, is the existing width enough to display the information? 
Can you test three stages running at the same time?

Reply:
OK

Reply:
![image](https://github.com/apache/kyuubi/assets/51512358/745ef6de-da1a-4efc-acd2-9583843d3995)
your guess is right, when there have 3 stages at the same time, the log looks not correct, like the screenshot, the first is 3 stages at the same time, the scond is 1 stage, the width from 
`val width = TerminalWidth / stages.size`
`private val TerminalWidth = sys.env.getOrElse("COLUMNS", "80").toInt`
i think we can expand the default TerminalWidth
`[There is no job about this stage] `add length is this

Reply:
i use the debug to test the width
1. this show the lastProgressbar show there only 1 stage, the bar show there have 3 stages
![image-20231024104430257](https://github.com/apache/kyuubi/assets/51512358/ac435c2f-c336-41e4-a5d3-eefa6171ea0b)

2. our terminalWidth default is 80, i think it's the best width to show progress info, the follow show when we make the default terminalWith is 180, it will show too long
![image-20231024104935906](https://github.com/apache/kyuubi/assets/51512358/f609e613-5ffd-43d5-9c92-64b5b1027006)

there are 2 advice to improvement this
1. only display 1 stage at the same time
2. the not improvement logic is. 
```
// length 9
val header = s"[Stage ${s.stageId}:"
// length 12~15
val tailer = s"(${s.numCompleteTasks} + ${s.numActiveTasks}) / $total]"
// if there have 3 stages, the width is 27. it will show 3~6 precent info
val w = width - header.length - tailer.length
```
we can add jobHeader.length in the expr `val w = width - header.length - tailer.length + jobHeader.lenght`, because we add job Info, so i think it's the best way
![image](https://github.com/apache/kyuubi/assets/51512358/d20f985d-4fac-4f76-8290-63c726767c27)
just like the photo , the bar show 3 stages at the same time, the lastProgressBar show only 1 stage at the same time

what do you think? @wForget @cxzl25